js/common/lib/tensor.ts

```diff
@@ -284,7 +286,7 @@ export interface TensorFromImageOptions {
    */
   dataType?: 'float32'|'uint8';
   /**
-   * Tensor channel layout - default is 'NHWC'
+   * Tensor channel layout - default is 'NCHW' - TODO: add support for 'NHWC'
    */
```

Comment:
for `TODO: add support for 'NHWC'`:

add code in implementation when tensorLayout is NHWC throws exception